x-pack/test/security_solution_cypress/cypress/tasks/login.ts

```diff
@@ -106,6 +106,14 @@ export const constructUrlWithUser = (user: User, route: string) => {
   return builtUrl.href;
 };
 
+const getProviderName = () => {
+  if (Cypress.env('CLOUD_SERVERLESS')) {
+    return 'cloud-basic';
+  } else {
+    return 'basic';
+  }
+};
+
```

Comment:
__note/issue:__ would you mind switching to the approach I'm using here, so that the tests can work in any current and future environments, no matter how providers are configured? https://github.com/elastic/kibana/pull/165810/files#diff-c20246118002c8381fa4a7b666148fcea8e1a68afa0c3d3100fe362b5e40ebb2R117-R134